src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -424,10 +424,8 @@ def geoprocessing_chains(aoi, wkaoi, exchange, errback, choose_worker):
     ]
 
     return [
-        start.s(opname, data, wkaoi).set(exchange=exchange,
-                                         routing_key=worker) |
-        finish.s().set(exchange=exchange,
-                       routing_key=worker) |
+        run.s(opname, data, wkaoi).set(exchange=exchange,
+                                       routing_key=worker) |
```

Comment:
I don't think that we need any of the `exchange` and `routing_key` stuff now that SJS is out of the picture. The main reason for it was to pin tasks to the worker where things started so that they'd finish there too. It may be another task, but probably worth doing.